src/System.Device.Gpio/System/Device/Gpio/LibgpiodDriverEventHandler.Linux.cs

```diff
@@ -32,7 +34,7 @@ public LibGpiodDriverEventHandler(int pinNumber, SafeLineHandle safeLineHandle)
 
         private void SubscribeForEvent(SafeLineHandle pinHandle)
         {
-            int eventSuccess = Interop.libgpiod.gpiod_line_request_both_edges_events(pinHandle, $"Listen {_pinNumber} for both edge event");
+            int eventSuccess = Interop.libgpiod.gpiod_line_request_both_edges_events(pinHandle, consumerName);
```

Comment:
I'm wondering if this should be some combination of old and new version. cc: @joperezr

Reply:
In my opinion this "listening ..." information is useless and somewhat replicated. First, we are not using the line info internally for any purpose. For the description information of the line info, it is already described if the pin is as input or output, there is no need to say it again in the consumer tag.

Reply:
I think I agree with @microhobby, I don't think that the info is all that useful and simply having the process name might be enough, plus as @microhobby pointed out, it does follow what libgpiod guys use for their utilities. I'm fine with keeping this just with the consumer name for now, and then if somebody asks for the more descriptive message then we can have a discussion on whether or not to add it back.

Reply:
@krwq if you have concerns about this, let me know what you think.

Reply:
Agree with @joperezr, plus looks more clean :), thank you for demonstration @microhobby